client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanFactory.java

```diff
@@ -44,24 +45,37 @@
  * Creates a new bean instance that mirrors its properties from Central Dogma.
  *
  * <p>In the following example, {@code mirroredFoo.getA()} and {@code getB()} will return the latest known
- * values retrieved from {@code /myProject/myRepo/foo.json} in Central Dogma:
+ * values retrieved from {@code /myProject/myRepo/foo.json} along with the latest revision in Central Dogma.
+ * If the latest values are not available yet, the revision will be set to null.
  *
  * <pre>{@code
- * > CentralDogmaFactory factory = new CentralDogmaFactory(dogma, new ObjectMapper());
- * > Foo mirroredFoo = factory.get(new Foo(), Foo.class);
- * >
- * > @CentralDogmaBean(project = "myProject",
- * >                   repository = "myRepo",
- * >                   path = "/foo.json")
- * > public class Foo {
- * >     private int a;
- * >     private String b;
- * >
- * >     public int getA() { return a; }
- * >     public void setA(int a) { this.a = a; }
- * >     public String getB() { return b; }
- * >     public void setB(String b) { this.b = b; }
- * > }
+ *  CentralDogmaFactory factory = new CentralDogmaFactory(dogma, new ObjectMapper());
```

Comment:
Could you add `>` in front of lines from 52 to 67 and replace `{@literal @}` with `@` please?
I was asking you just remove the `>` from 73 to 75 lines.
The reason why we used `>` here is java-doc misinterpret when the line in `{@code }` block starts with `@`.
For example:
```
{@ code
...
@override
...
}
```
Although all of the codes in `{@code }` are displayed as they are, the above case is not working as we expect.
So we have decided to put `>` at the front of every line when there is a line which starts with `@` in the `{@code }` block.
The `>` is removed when the gradle builds, so we can see the the clean java-docs. :-)